test/factories/data_center_stats.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :data_center_stat do
+    gossip_nodes_count { 1 }
+    validators_count { 1 }
+    data_center_id { create(:data_center).id }
```

Comment:
It can be done in a simpler way
https://github.com/thoughtbot/factory_bot/blob/main/GETTING_STARTED.md#associations